src/core/contexts/abstractwindowcontext.cpp

```diff
@@ -126,7 +126,7 @@ namespace QWK {
             return false;
         }
 
-        for (auto item : m_hitTestVisibleItems) {
+        for (const auto &item : m_hitTestVisibleItems) {
```

Comment:
如果你要改这一行的话，我是建议改成这样：
```suggestion
        for (auto&& item : std::as_const(m_hitTestVisibleItems)) {
```